code-injection-analysis/InitVigilance_jamieDSOTesting/CWE_079_094_116.js

```
@@ -0,0 +1,161 @@
+          .replace(/'/g, "&apos;");
+};
+
+module.exports.decode = function(s) {
+  return s.replace(/&amp;/g, "&")
+          .replace(/&quot;/g, "\"")
+          .replace(/&apos;/g, "'");
+};
+
+// should trigger codeql/javascript/ql/src/Security/CWE-116/IncompleteHtmlAttributeSanitization.ql
+
+var app = require('express')();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'app'. Consider to delete it or rename it.

'app' is defined here:


(lint/suspicious/noRedeclare)